src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -545,6 +550,7 @@ namespace NServiceBus
     }
     public interface IMessageHandlerContext : NServiceBus.IBusContext, NServiceBus.IMessageProcessingContext
     {
+        NServiceBus.Persistence.SynchronizedStorageSession SynchronizedStorageSession { get; }
```

Comment:
Users can have their storages. The name here was meant to stress out the fact that this storage is synchronized with message handling context. But maybe the solve fact that it is a property of a message handling context is enough?